alternator/stats.hh

```diff
@@ -75,6 +76,9 @@ public:
         utils::timed_rate_moving_average_summary_and_histogram batch_write_item_latency;
         utils::timed_rate_moving_average_summary_and_histogram batch_get_item_latency;
         utils::timed_rate_moving_average_summary_and_histogram get_records_latency;
+
+        utils::estimated_histogram batch_get_item_sizes{30};
+        utils::estimated_histogram batch_write_item_sizes{30};
```

Comment:
Note that there is no reason why you need to reach 400, the limit of number of items in a batch is 100. Does this affect the number "30" chosen here?
Please add a comment explaining what you did in the commit message on what this 30 means. Where does the number 1 and 1.2 in the commit message come from? Are those the defaults of estimated_histogram?

Reply:
1 and 1.2 are the defaults for estimated_histogram. estimated_histogram also rounds to the default integer, which is why it works well for low integer numbers.
If 100 is enough, I can use 20 buckets:
1	2	3	4	5	6	8	10	12	15	18	22	27	33	40	48	58	70	84	101